msfdb

```diff
@@ -7,4 +7,76 @@
 require 'pathname'
 require Pathname.new(__FILE__).realpath.expand_path.parent.join('config', 'boot')
 require 'msf/core/db_manager/http/http_db_manager_service'
-HttpDBManagerService.new().start(:Port => '8080', :Host => '0.0.0.0')
\ No newline at end of file
+require 'optparse'
+
+class HelpError < StandardError; end
+
+class SwitchError < StandardError
+  def initialize(msg="Missing required switch.")
+    super(msg)
+  end
+end
+
+def parse_args(args)
+  opts = {}
+  opt = OptionParser.new
+  banner = "msfdb - A remote database process for Metasploit Framework.\n"
+  banner << "Usage: #{$0} [options] <var=val>"
+  opt.banner = banner
+  opt.separator('')
+  opt.separator('Options:')
+
+  # Defaults:
+  opts[:interface] = '0.0.0.0'
+  opts[:port] = 8080
+  opts[:ssl] = false
+  opts[:ssl_cert] = nil
+  opts[:ssl_key] = nil
+
+  opt.on('-i', '--interface  <interface>', String, 'Interface to listen on') do |p|
+    opts[:interface] = p
+  end
+
+  opt.on('-p', '--port       <port number>', Integer, 'Port to listen on') do |p|
+    opts[:port] = p
+  end
+
+  opt.on('-s', '--ssl', 'Enable SSL on the server') do |p|
+    opts[:ssl] = true
+  end
+
+  opt.on('-c', '--cert      <path/to/cert.pem>', String, 'Path to your SSL Certificate file') do |p|
+    opts[:ssl_cert] = p
+  end
+
+  opt.on('-k', '--key       <path/to/key.pem>', String, 'Path to your SSL Key file') do |p|
+    opts[:ssl_key] = p
+  end
+
+  opt.on_tail('-h', '--help', 'Show this message') do
```

Comment:
Modify description to "Show this help information" to be consistent with other commands.